covalent_slurm_plugin/slurm.py

```diff
@@ -45,19 +45,21 @@
     "username": "",
     "address": "",
     "ssh_key_file": "",
+    "sshproxy": {},
     "cert_file": None,
     "remote_workdir": "covalent-workdir",
     "create_unique_workdir": False,
-    "slurm_path": None,
-    "conda_env": None,
+    "conda_env": "base",
```

Comment:
I'm not sure we should make this the default.  This implies if you don't use Conda at all, you still need to work with the `conda_env` variable.  Since Conda purely extends functionality, I'd prefer `""` for the default.

Reply:
Makes sense. I had used "base" simply to make it backwards-compatible, but I don't think that's the best option.